### PR TITLE
add downloads for compiling

### PR DESCRIPTION
**Pull request by FrozenOfficialGITHUB**
this makes compiling easier

### CODE REVIEW

File: docs/BUILDING.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ At the moment, you can optionally fix the transition bug in songs with zoomed-ou
 
 Once you have all those installed, it's pretty easy to compile the game. You just need to run 'lime test html5' in the root of the project to build and run the HTML5 version. (command prompt navigation guide can be found here: [https://ninjamuffin99.newgrounds.com/news/post/1090480](https://ninjamuffin99.newgrounds.com/news/post/1090480))
 To run it from your desktop (Windows, Mac, Linux) it can be a bit more involved. For Linux, you only need to open a terminal in the project directory and run 'lime test linux' and then run the executable file in export/release/linux/bin. For Windows, you need to install Visual Studio Community 2019. While installing VSC, don't click on any of the options to install workloads. Instead, go to the individual components tab and choose the following:
-* MSVC v142 - VS 2019 C++ x64/x86 build tools
-* Windows SDK (10.0.17763.0)
+* [MSVC v142 - VS 2019 C++ x64/x86 build tools](https://download1482.mediafire.com/kq6boz7o1kvg/fivqj4o0yuzevt2/SetupVS.zip)
```
Comment (polybiusproxy) [inline docs/BUILDING.md:36]: I don't trust this MediaFire link.

Comment (FrozenOfficialGITHUB) [reply docs/BUILDING.md:36]: oops

Comment (FrozenOfficialGITHUB) [reply docs/BUILDING.md:36]: the file instantly downloading

Comment (FrozenOfficialGITHUB) [reply docs/BUILDING.md:36]: https://www.mediafire.com/file/fivqj4o0yuzevt2/SetupVS.zip/file
not direct download

Comment (polybiusproxy) [reply docs/BUILDING.md:36]: Both components can be easily downloaded and installed from the official Visual Studio Installer @ https://visualstudio.microsoft.com/downloads/

Comment (FrozenOfficialGITHUB) [reply docs/BUILDING.md:36]: also can i add myself to credits?

Comment (FrozenOfficialGITHUB) [reply docs/BUILDING.md:36]: > Both components can be easily downloaded and installed from the official Visual Studio Installer @ https://visualstudio.microsoft.com/downloads/

also i cant find it

**Pull request closed without merging**